lib/streamlit/elements/arrow.py

```diff
@@ -82,12 +82,12 @@ def dataframe(
             pandas styling features, like bar charts, hovering, and captions.
 
         width : int or None
-            Desired width of the dataframe expressed in pixels. If None, the width
-            will be automatically calculated based on the column content.
+            Desired width of the dataframe expressed in pixels. If None (default),
+            selects the width automatically.
 
         height : int or None
-            Desired height of the dataframe expressed in pixels. If None, a
-            default height is used.
+            Desired height of the dataframe expressed in pixels. If None (default),
+            selects the height automatically.
 
         use_container_width : bool
             If True, set the dataframe width to the width of the parent container.
```

Comment:
```suggestion
            Whether to override ``width`` with the width of the parent container. The default is ``False``.
            If ``use_contatiner_width`` is ``True``, Streamlit sets the width of the dataframe to match the
            width of the parent container.
```
To follow through with my above comment, this argument could be something like this. (It didn't explicitly state its default before.) Note that the comment isn't letting me include next line. I'm proposing the "overrides" language in place of "takes precedence." GitHub wouldn't let me include line 94 in the comment. :)